locales/de-DE.json

```diff
@@ -367,7 +367,7 @@
   "share-target": {
     "description": "Elk kann so konfiguriert werden, dass du Inhalte aus anderen Anwendungen teilen kannst, installiere einfach Elk auf deinem Gerät oder Computer und melden dich an.",
     "hint": "Um Inhalte mit Elk zu teilen, muss Elk installiert sein und du musst angemeldet sein.",
-    "title": "Teile via Elk"
+    "title": "Teile mit Elk"
```

Comment:
that's not correct, pls revert
or provide an alternative

Reply:
Du teilst nicht mit Elk, sondern du teilst mit dem gesamten Mastodon Universum bzw du teils den Link extern jemandem mit
Aber hier teilst du einen Link oder Post or whatever via Elk, denn du benutzt die App Elk um es zu teilen

Reply:
what about "teile über Elk" ?